parlai/agents/seq2seq/seq2seq.py

```diff
@@ -73,7 +73,7 @@ def add_cmdline_args(argparser):
                            help='learning rate')
         agent.add_argument('-dr', '--dropout', type=float, default=0.1,
                            help='dropout rate')
-        agent.add_argument('-clip', '--gradient-clip', type=float, default=-1,
+        agent.add_argument('-clip', '--gradient-clip', type=float, default=0.1,
```

Comment:
changing defaults is dangerous because people have scripts that they run and now they are different.
would advise leaving same unless there's really good reason to

Reply:
-1 seemed to clearly work worse (immediate breakage on the twitter training without clipping) so I wanted to update this to reflect, but I can leave it at -1

Reply:
let's check existing scripts at least